src/gba/overrides.c

```diff
@@ -424,6 +424,23 @@ void GBAOverrideApplyDefaults(struct GBA* gba, const struct Configuration* overr
 			GBAOverrideApply(gba, &override);
 		} else if (GBAOverrideFind(overrides, &override)) {
 			GBAOverrideApply(gba, &override);
+		} else {
+			// Apply Everdrive UTTD save detection
+			switch (override.id[0]) {
+			case '1':
+				override.savetype = SAVEDATA_EEPROM;
+				break;
+			case '2':
+				override.savetype = SAVEDATA_SRAM;
+				break;
+			case '3':
+				override.savetype = SAVEDATA_FLASH512;
+				break;
+			case '4':
+				override.savetype = SAVEDATA_FLASH1M;
+				break;
```

Comment:
I'd recommend a `default: return;` here so it doesn't call GBAOverrideApply if it's not one of those four, but otherwise looks good.

Reply:
Ah actually it's not compiling in the CI. I might have changed the constants to start with GBA_* in the meantime.